riscv/csrs.h

```diff
@@ -256,6 +256,20 @@ class mstatus_csr_t final: public base_status_csr_t {
 
 typedef std::shared_ptr<mstatus_csr_t> mstatus_csr_t_p;
 
+class mnstatus_csr_t final: public csr_t {
```

Comment:
This could be `basic_csr_t` and only override `unlogged_write()`. That would be more consistent with the other simple CSRs.

Reply:
Thanks for the suggestion.  I had forgotten about `basic_csr_t`.